lib/config/presets/local/index.ts

```diff
@@ -21,17 +22,32 @@ export function getPreset({
   }
   switch (platform.toLowerCase()) {
     case PLATFORM_TYPE_GITLAB:
-      return gitlab.getPresetFromEndpoint(pkgName, presetName, endpoint);
+      return gitlab.getPresetFromEndpoint(
+        pkgName,
+        presetName,
+        presetPath,
+        endpoint
+      );
     case PLATFORM_TYPE_GITHUB:
-      return github.getPresetFromEndpoint(pkgName, presetName, endpoint);
+      return github.getPresetFromEndpoint(
+        pkgName,
+        presetName,
+        presetPath,
+        endpoint
+      );
     case PLATFORM_TYPE_BITBUCKET_SERVER:
       return bitbucketServer.getPresetFromEndpoint(
         pkgName,
         presetName,
         endpoint
       );
```

Comment:
This should support paths too. Why it's not implemented?

Should also support it, see [api](https://docs.atlassian.com/bitbucket-server/rest/7.10.0/bitbucket-rest.html#idp218)

https://github.com/renovatebot/renovate/blob/424fc9ce498bba08571fb30f023f41ef769a494e/lib/config/presets/bitbucket-server/index.ts#L20

Reply:
Waiting for an answer.

Reply:
Fixed